.github/workflows/test-mobile.yml

```diff
@@ -42,4 +41,141 @@ jobs:
       - name: Run linter
         run: pnpm lint --filter="live-mobile" --api="http://127.0.0.1:${{ steps.turborepo-cache-server.outputs.port }}" --token="yolo" --team="foo"
       # - name: Run code checkers
-      #   run: pnpm typecheck --filter="live-mobile" --api="http://127.0.0.1:${{ steps.turborepo-cache-server.outputs.port }}" --token="yolo" --team="foo"
+      #   run: pnpm typecheck --filter="live-mobile" --api="http://127.0.0.1:9080" --token="yolo" --team="foo"
+  
+  detox-tests-ios:
```

Comment:
So we need to split this in a different workflow so we can run it only on nightly run or something as they take to long to succeed for now.

Reply:
You mean one each for the linter, one for Detox iOS and one for Detox Android?

Reply:
I've split these into a codecheck workflow and a detox test workflow. If it looks ok to you I will proceed and remove the 'run on push' and replace it with the cron job to run nightly